tests/unit/test_tf_estimator.py

```diff
@@ -305,6 +308,49 @@ def test_create_model_with_optional_params(sagemaker_session):
     assert model.vpc_config == vpc_config
 
 
+@patch("sagemaker.tensorflow.estimator.TensorFlow._create_tfs_model")
```

Comment:
You should avoid patching a private method from within the class that you are testing to not lock the method signature. You can mock using:

```python
sagemaker.tensorflow.estimator.TensorFlow.create_model
```

instead.